tarsier/core.py

```diff
@@ -77,3 +98,15 @@ async def _remove_tags(adapter: BrowserAdapter) -> None:
         script = "return window.removeTags();"
 
         await adapter.run_js(script)
+
+    @staticmethod
+    async def _hide_non_tag_elem(adapter: BrowserAdapter) -> None:
```

Comment:
Should maybe just use the same name as TS (Should be done for all similar methods) 
```suggestion
    async def _hide_non_tag_elements(adapter: BrowserAdapter) -> None:
```